phpBB/includes/acp/acp_extensions.php

```diff
@@ -65,22 +77,22 @@ function main()
 		* @var	string	ext_name		Extension name from request
 		* @var	int		safe_time_limit	Safe limit of execution time
 		* @var	int		start_time		Start time
+		* @var	string	tpl_name		Template file to load; leave empty to continue execution, filled in if ready to finish
```

Comment:
Additional var to allow execution control: if ready to move to next action, it should be set from the extension.  Empty means continue executing here.